file/sst_file_manager_impl.cc

```diff
@@ -450,7 +451,6 @@ void SstFileManagerImpl::OnAddFileImpl(const std::string& file_path,
     // File was added before, we will just update the size
     total_files_size_ -= tracked_file->second;
     total_files_size_ += file_size;
-    cur_compactions_reserved_size_ -= file_size;
```

Comment:
Removing this seems appropriate to me. I can't think of any reason why we'd want to decrement `cur_compactions_reserved_size_` here. We would come down this path only in a non-compaction case.

Reply:
Thank you for spending time to help check this!